ecommerce/frontend/src/components/checkout/CreditCard.less

```diff
@@ -0,0 +1,22 @@
+.creditcard-container {
+    padding: 16px;
+    display: inline-block;
+    &__selected {
+        border: 2px solid gainsboro;
+    }
```

Comment:
border moves card by 4px in total, so assign transparent border even when not selected